pkg/machine/keys.go

```diff
@@ -78,13 +78,14 @@ func generatekeys(writeLocation string) error {
 	if err := cmd.Start(); err != nil {
 		return err
 	}
+	errMsg, ioErr := io.ReadAll(stdErr)
```

Comment:
it seems it would be much easier to use
```
stderr := &bytes.Buffer{}
cmd.Stderr = stderr
```
and then just do `strings.TrimSpace(stderr.String())` to get the error message
instead of calling `cmd.StderrPipe()`

Reply:
Taking this suggestion, thanks!  PTAnotherL